bucket/codeql.json

```diff
@@ -0,0 +1,22 @@
+{
+    "version": "2.10.1",
+    "description": "Source code security analyzer from GitHub",
+    "homepage": "https://github.com/github/codeql-cli-binaries",
+    "license": "MS-PL",
```

Comment:
Where is this mentioned? I couldn't find it here: https://github.com/github/codeql-cli-binaries/blob/main/LICENSE.md

Reply:
Agree, which `license` value should I use for https://github.com/github/codeql-cli-binaries/blob/main/LICENSE.md?

Reply:
fixed on https://github.com/ScoopInstaller/Main/pull/3771/commits/34ac0b6d2935cfabaa9d18892d316a551caf056f